### ISSUE TITLE
prefixItems should not allow arrays of variable length

### ISSUE THREAD

**Issue by markoutso** [json-schema-org/understanding-json-schema#191]

**Comment by jdesrosiers** [comment]
I agree "tuple" isn't a perfect description, but I think it's more helpful than misleading. `prefixItems` (or previously array-form-`items`) is primarily useful for describing a tuple. I think the way it's written makes it sound like, if you're using this keyword then you're describing a tuple, which isn't always true. I think I'd like the example to be a tuple and call it a tuple, but make it more clear that you can describe things that are not quite tuples with this keyword as well.

In any case, I have another three months before I'll have a chance to work on this. So, don't wait on me if anyone wants to put up a PR.

**Comment by gregsdennis** [comment]
> is **_primarily_** useful for describing a tuple

I think this is an incorrect assumption, and probably the basis of this confusion.
